packages/sit-onyx/package.json

```diff
@@ -55,5 +55,8 @@
     "sass": "catalog:",
     "vue": "catalog:",
     "vue-component-type-helpers": "^2.1.6"
+  },
+  "dependencies": {
+    "@storybook/addon-a11y": "^8.3.4"
```

Comment:
Please install the storybook add-on as *devDependencies* (using `pnpm i -D @storybook/addon-a11y`).
If a package is listed in the *dependencies*, it will be installed when `sit-onyx` itself is installed by someone. We don't want that as this addon is only used for development and build time.